lms/templates/certificates/_accomplishment-footer.html

```diff
@@ -1,16 +1,17 @@
+<%page expression_filter="h"/>
 <div class="wrapper-footer">
 
     <footer class="footer-app" role="contentinfo" id="company-info">
         <div class="footer-app-copyright">
-            <p class="copy copy-micro">${copyright_text}</p>
+            <p class="copy copy-micro">${copyright_text | n, decode.utf8 }</p>
```

Comment:
Why is this exempt from the HTML filter?

Reply:
Because the copyright was not properly appearing. Html character entities was getting escaped so had to remove escaping.
```
context['copyright_text'] = u'&copy; {year} {platform_name}. {reserved}.'.format(
        year=datetime.now(pytz.timezone(settings.TIME_ZONE)).year,
        platform_name=platform_name,
        reserved=reserved
    )
```

Reply:
Does the XSS linter flag this? You might need to explicitly disable it for this line.